taxonomy/migrations/0022_merge_20220819_0834.py

```diff
@@ -0,0 +1,14 @@
+# Generated by Django 3.2.12 on 2022-08-19 08:34
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('taxonomy', '0021_recreate_programskill'),
+        ('taxonomy', '0021_refreshprogramskillsconfig'),
+    ]
+
+    operations = [
+    ]
```

Comment:
nit: you can remove your migration and recreate to not have this merge migration.